src/OrbitQt/HistogramWidget.h

```diff
@@ -20,6 +20,14 @@
 #include "App.h"
 #include "Statistics/Histogram.h"
 
+namespace orbit_histogram_widget {
```

Comment:
Please follow the rules for namespace outlined in https://github.com/google/orbit/blob/main/DEVELOPMENT.md#namespaces. I know `OrbitQt` doesn't always have the `orbit_qt` namespace (as it's a very very old module), but let's not add new ones that don't follow our rules.

Reply:
I see, but this way `GenerageBinWidth` will be in the `orbit_qt` too. Is that OK?

Reply:
Or you can make it a `static` function of `HistogramWidget`, maybe that's even better.

Reply:
> Or you can make it a static

But the styleguide seems to discourage this "Static methods of a class should generally be closely related to instances of the class or the class's static data".

Reply:
You can't say it's closely related because `DrawHistogram` is a static free function. So I guess `GenerateHistogramBinWidth` should also be a free function. But everything could just as well be static methods, I can't see how they could be any more closely related.